### DISCUSSION TITLE
You should say it requires React 18

### DISCUSSION THREAD

**Discussion by tombohub** [chakra-ui/chakra-ui#6123] [Ideas]
In Getting Started it doesn't say that version 2.1.0 requires React 18.

Remix by default comes with React 17. I have spent time to find out why is not working.
Please state explicitly what is needed to run the specific Chakra version in Getting Started page , Thank you

**Comment by simoncollins** [comment]
@tombohub did you manage to get things working with React 18 and 2.1.0? I'm getting hydration errors.

**Reply by tombohub** [reply]
@simoncollins I have only played with it and everything was working fine. Are you using Remix or something else?

**Reply by simoncollins** [reply]
Yes, I'm trying to get Remix working with Chakra 2.10. When I make the corresponding changes to output of the generator to support React 18: https://github.com/remix-run/remix/pull/2540#issuecomment-1125770860 I get runtime issues hydrating when following the guide here: https://chakra-ui.com/guides/getting-started/remix-guide

Was just wondering if you have the same setup or if you reverted to a version of Chakra that supports React 17.

**Reply by tombohub** [reply]
I don't think I've used older version of Chakra because I know I had errors before I realized I need React 18 and then was working fine. I was also looking those 2 examples at the end. Maybe I took some code from there too. I wish I have template now but i have deleted it. 

Also, since then I have learned that emotion 11 works with ssr out of the box. Cache and other things they instruct you to do is only if you need nth child selectors.
